testsuite/features/build_validation/smoke_tests/smoke_tests.template

```diff
@@ -214,17 +222,23 @@ Feature: Smoke tests for <client>
 
 @skip_for_sle_micro_ssh_minion
 @sle_micro_minion
+@skip_for_sl_micro
   # workaround for SLE Micro minion issue bsc#1209374
+  # WORKAROUND for SL Micro because of bsc#1227451 and bsc#1229567
   Scenario: Enable and start Prometheus Node Exporter service
     And I start the "prometheus-node_exporter.service" service on "<client>"
     And I enable the "prometheus-node_exporter.service" service on "<client>"
 
 @skip_for_sle_micro_ssh_minion
+@skip_for_sl_micro
+  # WORKAROUND for SL Micro because of bsc#1227451 and bsc#1229567
   Scenario: Visit Node monitoring endpoint on the <client>
     When I wait until "node" exporter service is active on "<client>"
     And I visit "Prometheus node exporter" endpoint of this "<client>"
 
 @skip_for_sle_micro
+@skip_for_sl_micro
+  # WORKAROUND for SL Micro because of bsc#1227451 and bsc#1229567
```

Comment:
The skip looks good to me, but that's unrelated to the commented bugs.
Apache and Postgres components are not meant to be installed directly on SL Micro, so that's the reason why we don't support these exporters in the product, that's unrelated to Node Exporter which just collects HW metrics.